src/GraphQL.StarWars/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[*.cs]
+resharper_virtual_member_call_in_constructor_highlighting = none
```

Comment:
This is to avoid [Virtual member call in constructor](https://www.jetbrains.com/help/resharper/VirtualMemberCallInConstructor.html) warning by Resharper
![image](https://github.com/graphql-dotnet/graphql-dotnet/assets/4518483/b9c05f34-b273-48c3-9c5b-244c22d273fd)